packages/bpk-component-navigation-bar/src/BpkNavigationBar.tsx

```diff
@@ -34,6 +36,8 @@ export type BarStyle = (typeof BAR_STYLES)[keyof typeof BAR_STYLES];
 export type Props = {
   id: string;
   title: ReactNode;
+  titleTextStyle?: TextStyle;
```

Comment:
Make it easy for consumers to control the markup without needing to do their own title layout